lib/mayaUsd/ufe/UsdTransform3dMatrixOp.cpp

```diff
@@ -433,8 +436,19 @@ UsdTransform3dMatrixOpHandler::editTransform3d(const Ufe::SceneItem::Ptr& item)
     });
     bool foundMatrix = (i != xformOps.end());
 
-    return foundMatrix ? UsdTransform3dMatrixOp::create(usdItem, *i)
-                       : _nextHandler->editTransform3d(item);
+    // If we've found a matrix op, but there is a more local non-matrix op in
+    // the stack, the more local op should be used to handle the edit.
+    bool moreLocalNonMatrix = foundMatrix ?
+        (std::find_if(i, xformOps.end(), [](const UsdGeomXformOp& op)
+        { return op.GetOpType() != UsdGeomXformOp::TypeTransform; }) 
+         != xformOps.end()) : false;
+
+    // We can't handle pivot edits, so in that case pass on to the next handler.
+    return
+        (foundMatrix && !moreLocalNonMatrix && 
+         (hint.type() != Ufe::EditTransform3dHint::EditPivot)) ? 
```

Comment:
Use the EditTransform3dHint to pass on the edit to the next Transform3dHandler, if a pivot edit.